var/spack/repos/builtin/packages/boxlib/package.py

```diff
@@ -25,26 +25,33 @@
 from spack import *
 
 
-class Boxlib(Package):
+class Boxlib(CMakePackage):
     """BoxLib, a software framework for massively parallel
        block-structured adaptive mesh refinement (AMR) codes."""
 
     homepage = "https://ccse.lbl.gov/BoxLib/"
-    url = "https://ccse.lbl.gov/pub/Downloads/BoxLib.git"
+    url = "https://github.com/BoxLib-Codes/BoxLib.git"
```

Comment:
I think it would be good to use an actual download tarball for the URL. There seem to be plenty for this repository. The latest is:
https://github.com/BoxLib-Codes/BoxLib/archive/16.12.2.tar.gz